main/src/main/scala/sbt/internal/Load.scala

```diff
@@ -1039,7 +1039,7 @@ private[sbt] object Load {
         // Grab all the settings we already loaded from sbt files
         def settings(files: Seq[File]): Seq[Setting[_]] = {
           if (files.nonEmpty)
-            log.info(s"${files.map(_.getName).mkString("Loading settings from ", ",", " ...")}")
+            log.info(s"${files.map(_.getPath).mkString("Loading settings from ", ",", " ...")}")
```

Comment:
The expectation set by @dwijnand says:

```
[info] Loading settings from build.sbt ...
[info] Loading settings from akka-actor/build.sbt ...
[info] Loading settings from akka-actor-tests/build.sbt ...
[info] Loading settings from akka-agent/build.sbt ...
```

so that should be relative paths from the root of the build, not the entire absolute path.

Reply:
I couldn't find a straight-forward way to wire the build root path to make it relative to it.

What do you think of this?

```scala
-            log.info(s"${files.map(_.getName).mkString("Loading settings from ", ",", " ...")}")
+            log.info(s"${files.map(_.getName).mkString(s"Loading settings for project ${p.id} from ", ",", " ...")}")
```

output:

```
[info] Loading settings for project global-plugins from idea.sbt,global-plugins.sbt ...
[info] Loading global plugins from /Users/dnw/.dotfiles/.sbt/1.0/plugins
[info] Loading settings for project t-multi-project-build-sbts-build from plugins.sbt ...
[info] Loading project definition from /s/t-multi-project-build.sbts/project
[info] Loading settings for project root from build.sbt ...
[info] Loading settings for project baz from build.sbt ...
[info] Loading settings for project bar from build.sbt ...
[info] Loading settings for project foo from build.sbt ...
[info] Set current project to root (in build file:/s/t-multi-project-build.sbts/)
```

Reply:
@dwijnand Sounds like a good middle ground.

Reply:
thanks for the hint @dwijnand